invenio_app_ils/records/serializers/access.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```

Comment:
I am not sure I understand this serializer. If I provide my own implementation of the factory, I might not return patrons at all...